pkg/cache/repository.go

```diff
@@ -131,14 +132,15 @@ func (cr *Repository) Invalidate(label, runtimeID string) error {
 }
 
 func (cr *Repository) InvalidateByID(id int64) error {
-	dbOps := func() error {
+	dbOps := func(tx *db.TxConnection) error {
```

Comment:
as above.. bug in code. TX should happen first and rolled back if Cache-invalidation fails :/

Reply:
Fixed